src/lib/pipeline_configuration.py

```diff
@@ -420,8 +420,16 @@ def clean_age_with_range_filter(text):
                            coded_field="age_coded",
                            analysis_file_key="age",
```

Comment:
On other projects we have removed this line, so that analysis files only include age_category, not age. But not sure if that's desirable, it may be helpful e.g. as a check or if anyone wants to look at the underlying age distribution more closely, so check if in doubt.

Reply:
Thanks, I will leave it and remove on request.